.github/workflows/_binary-build-linux.yml

```diff
@@ -78,7 +78,7 @@ on:
 jobs:
   build:
     runs-on: ${{ inputs.runs_on }}
-    timeout-minutes: 210
```

Comment:
If inputs.timeout-minutes is used, then the above should have a timeout-minutes variable, right? under "runs_on".

Reply:
Might be it! Simple definition got lost in many commits. Could you give it a try?

Reply:
Turns out need to update the type to "number" instead of "string".